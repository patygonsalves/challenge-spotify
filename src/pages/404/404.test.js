import React from 'react'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import PageNotFound from '.'

describe('<PageNotFound />', () => {
  test('should render PageNotFound', () => {
    const { container } = render(
      <PageNotFound />
    )

    expect(screen.getByText(/ops, parece que esta página não foi encontrada!/i)).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
