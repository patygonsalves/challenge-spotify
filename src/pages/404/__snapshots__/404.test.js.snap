// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> should render PageNotFound 1`] = `
<main
  class="notfoundX-404"
>
  Ops, parece que esta página não foi encontrada!
</main>
`;
